SimpleBrowser.UnitTests/SimpleBrowser.UnitTests.csproj

```diff
@@ -7,9 +7,9 @@
     <NeutralLanguage>en-US</NeutralLanguage>
     <VersionPrefix>1.0.0</VersionPrefix>
     <Authors>Nathan Ridley, Teun Duynstee, Kevin Yochum, Joe Feser</Authors>
-    <TargetFrameworks>net40</TargetFrameworks>
+    <TargetFrameworks>net45;netcoreapp2.0</TargetFrameworks>

```

Comment:
Should this be net45 or net40? In the context of this code review, I think should still be net40 - although an upgrade to net45 is imminent. Is there something in the unit test project that requires net45?

I apologize for sending comments only once a week. I'm working on a few different things and am going through a bit of a life change over the last four months. I'm a bit scattered. :)

Reply:
The actual library is targetting .NET 4.0; however the unit test project needs to be .NET 4.5 because of the NUnit 3 Test runner.

Reply:
@kevingy 
No worries, take your time :-).